source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -15,26 +24,28 @@ content: |
 title: Create the Master Key
 ref: create-the-master-key
 content: |
-
-  The following diagram shows how the **master key** is created and stored
-  when using a KMS provider:
+  The following diagram shows the steps required to create a new
+  **master key** on a KMS provider.
 
   .. image:: /figures/CSFLE_Master_Key_KMS.png
```

Comment:
This image isn't rendering in the staged version (same for other tabs).

Reply:
Yeah unfortunately none of them do on docs-ecosystem because we're redirecting to /drivers. Have an open ticket for DOP.